internal/namespaces/rdb/v1/custom_user.go

```diff
@@ -75,3 +78,149 @@ func userListBuilder(c *core.Command) *core.Command {
 
 	return c
 }
+
+func userCreateBuilder(c *core.Command) *core.Command {
+	type rdbCreateUserRequestCustom struct {
+		*rdb.CreateUserRequest
+		GeneratePassword bool
+	}
+
+	return &core.Command{
+		Short:     `Create a user on a given instance`,
+		Long:      `Create a user on a given instance.`,
+		Namespace: "rdb",
+		Resource:  "user",
+		Verb:      "create",
+		// Deprecated:    false,
+		ArgsType: reflect.TypeOf(rdbCreateUserRequestCustom{}),
+		ArgSpecs: core.ArgSpecs{
+			{
+				Name:       "instance-id",
+				Short:      `UUID of the instance you want to create a user in`,
+				Required:   true,
+				Deprecated: false,
+				Positional: false,
+			},
+			{
+				Name:       "name",
+				Short:      `Name of the user you want to create`,
+				Required:   false,
+				Deprecated: false,
+				Positional: false,
+			},
+			{
+				Name:       "password",
+				Short:      `Password of the user you want to create`,
+				Required:   false,
+				Deprecated: false,
+				Positional: false,
+				OneOfGroup: "password",
+			},
+			{
+				Name:       "generate-password",
+				Short:      `Will generate a valid password for the user`,
+				Required:   false,
```

Comment:
Should we put this field as a default true?

Reply:
Yes